client/my-sites/plans/jetpack-plans/faq/index.tsx

```diff
@@ -99,7 +99,7 @@ const JetpackFAQ: FC = () => {
 					className="jetpack-faq__section"
 				>
 					{ translate(
-						"If your site's Backup storage limit is reached, your older backups will be deleted. Depending on the size of your site and your site's Backup storage limit, your site's backup retention period may be reduced down to 7 days of your most recent backups. You will still be able to restore existing backups, but new site updates will not be backed up until you free up storage or upgrade your storage limit."
+						"If your site's VaultPress Backup storage limit is reached, your older backups will be deleted. Depending on the size of your site and your site's VaultPress Backup storage limit, your site's backup retention period may be reduced down to 7 days of your most recent backups. You will still be able to restore existing backups, but new site updates will not be backed up until you free up storage or upgrade your storage limit."
```

Comment:
This answer to the FAQ reads a bit weird to me, imo..  I feel it is just answering the question about backup storage limits and retention and it just reads strange to me saying,
> "If your site's VaultPress Backup storage limit is reached, your older backups will be deleted. Depending on the size of your site and your site's VaultPress Backup storage limit,..

In this case I don't think we necessarily trying to refer the **brand** (VaultPress Backup) when referring to the sites backup storage limit.
I think in this case it should simply be:

> "If your site's backup storage limit is reached, your older backups will be deleted. Depending on the size of your site and your site's backup storage limit,..

I think maybe this happened because the FAQ had the word "Backup" incorrectly capitalized in the first place.

Reply:
That makes sense, I'll update it. Thanks!